Switch to Standard Term Support (STS) releases (#7517)
dotnet/core: release-notes/7.0/supported-os.md

# .NET 7 - Supported OS versions

> [.NET 7](README.md) is an [STS release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.

Reviewer: ```suggestion
[.NET 7](README.md) is a [standard-support release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```

> [.NET 7](README.md) is a [Standard Support release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.

Author: ```suggestion
[.NET 7](README.md) is a [Standard Term Support release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```
